simde/x86/svml.h

```diff
@@ -3661,6 +3616,91 @@ simde_mm512_mask_exp_pd(simde__m512d src, simde__mmask8 k, simde__m512d a) {
   #define _mm512_mask_exp_pd(src, k, a) simde_mm512_mask_exp_pd(src, k, a)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128
+simde_mm_cdfnorm_ps (simde__m128 a) {
+  #if defined(SIMDE_X86_SVML_NATIVE) && defined(SIMDE_X86_SSE_NATIVE)
+    return _mm_cdfnorm_ps(a);
+  #elif (SIMDE_NATURAL_VECTOR_SIZE > 0)
+
+    simde__m128 a1 = simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.254829592));
+    simde__m128 a2 = simde_mm_set1_ps(SIMDE_FLOAT32_C(-0.284496736));
+    simde__m128 a3 = simde_mm_set1_ps(SIMDE_FLOAT32_C(1.421413741));
+    simde__m128 a4 = simde_mm_set1_ps(SIMDE_FLOAT32_C(-1.453152027));
+    simde__m128 a5 = simde_mm_set1_ps(SIMDE_FLOAT32_C(1.061405429));
+    simde__m128 p = simde_mm_set1_ps(SIMDE_FLOAT32_C(0.3275911));
+    simde__m128 one = simde_mm_set1_ps(SIMDE_FLOAT32_C(1.0));
+
+    simde__m128 sign = a;
+
+    a = simde_mm_div_ps(simde_x_mm_abs_ps(a), simde_mm_sqrt_ps(simde_mm_set1_ps(SIMDE_FLOAT32_C(2.0))));
+
+    simde__m128 t = simde_mm_div_ps(one, simde_mm_add_ps(one, simde_mm_mul_ps(p, a)));
+    simde__m128 temp = simde_mm_add_ps(simde_mm_mul_ps(simde_mm_add_ps(simde_mm_mul_ps(simde_mm_add_ps(simde_mm_mul_ps(simde_mm_add_ps(simde_mm_mul_ps(a5, t), a4), t), a3), t), a2), t), a1);
+    simde__m128 y = simde_mm_sub_ps(one, simde_mm_mul_ps(simde_mm_exp_ps(simde_mm_mul_ps(a, (simde_x_mm_negate_ps(a)))),simde_mm_mul_ps(temp,t)));
+
+    simde__m128 r_ = simde_mm_mul_ps(simde_mm_set1_ps(SIMDE_FLOAT32_C(0.5)), simde_mm_add_ps(one, simde_x_mm_xorsign_ps(sign, y)));
```

Comment:
The `simde_x_mm_xorsign_ps` call is wrong; the arguments are reversed.

Reply:
This was the one causing failures:(